apps/team-awesome-web/src/components/time-entries/TimeEntries.tsx

```diff
@@ -43,7 +43,7 @@ interface TimeEntriesProps {
 export const TimeEntries = ({ ...props }: TimeEntriesProps) => {
```

Comment:
If you're only getting your prop values (`timeEntries` and `errorMessage`) from your `props` like `props.timeEntries` then there's no real advantage of spreading them here, you can just write `(props: PropsType)`

Reply:
This is a corner I've worked myself into (after advice from teachers): when I destructure `timeEntries` to be used as an initial state for `setTimeEntries` I wind up with conflicting declarations. Spreading `props` circumvents this and gives a `props.timeEntries` to be used in the code, making it _fairly_ well readable.